mixer/test/client/env/http_server.go

```diff
@@ -107,7 +107,7 @@ func handler(w http.ResponseWriter, r *http.Request, slowResponse bool) {
 	}
 	w.WriteHeader(http.StatusOK)
 	if slowResponse {
-		time.Sleep(2 * time.Second)
+		time.Sleep(3 * time.Second)
```

Comment:
how about client can use ?delay=3 to trigger it.  It is more flexible this way

Reply:
You don't even need a separate handler.

Reply:
Please take a look.